webpack.config.js

```diff
@@ -17,10 +15,16 @@ let config = {
     },
     port: 8080,
   },
+  entry: './docs/main.js',
   output: {
-    path: path.resolve(__dirname, 'dist'),
+    path: path.resolve(__dirname, 'docs'),
     filename: 'pushin.min.js',
   },
+  resolve: {
+    alias: {
+      pushin: path.resolve('./src'),
+    },
+  },
   optimization: {
     minimize: true,
     minimizer: [
```

Comment:
Nitpick: Now that this is essentially being used for development, maybe we should stop minimizing the JS and CSS?

Reply:
The JS is always minified by default when the mode is `production`.